### DISCUSSION TITLE
求教宝塔中安装了alist，后续升级使用什么命令呢？

### DISCUSSION THREAD

**Discussion by mangic** [AlistGo/alist#901] [Q&A]
求教宝塔中安装了alist（如图），后续升级使用什么命令呢？
![image](https://user-images.githubusercontent.com/33594145/162897351-800a3fef-fe55-4765-aec6-2e30b1c63bc2.png)

**Comment by xhofe** [comment]
用不上就不需要填写。

**Comment by mangic** [comment]
已解决，直接替换alist文件
